moto/ec2/responses/elastic_block_store.py

```diff
@@ -268,7 +268,7 @@ def reset_snapshot_attribute(self):
   <ownerId>{{ snapshot.owner_id }}</ownerId>
   <volumeSize>{{ snapshot.volume.size }}</volumeSize>
   <description>{{ snapshot.description }}</description>
-  <encrypted>{{ snapshot.encrypted }}</encrypted>
+  <encrypted>{{ 'true' if snapshot.encrypted else 'false' }}</encrypted>
```

Comment:
These changes were required to get the new tests to pass - I guess boto is more flexible in how it parses the response